README.md

```diff
@@ -43,9 +43,21 @@ out, running through [some tutorials](https://google.github.io/xls/tutorials/),
 [reporting bugs](https://github.com/google/xls/issues), and letting us know
 what you think!
 
+## Install using conda
+
+```console
+curl -O https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-x86_64.sh
+bash Miniconda3-py37_23.1.0-1-Linux-x86_64.sh -p conda-env/ -b
+source conda-env/bin/activate
+interpreter_main --version
```

Comment:
@cdleary this currently retusn `interpreter_main` is that expected?

Reply:
We've never done anything in particular to hook into that version flag output, must be what absl flags impl does by default.